Assets/Scripts/PlayerBase.cs

```diff
@@ -10,10 +10,10 @@ public class PlayerBase : MonoBehaviour
     private Rigidbody2D rb;
     private bool grounded;
 
-    public float horizontalMovementModifier = 0.25f;
-    public float jumpVelocity = 5.0f;
-    public float fallMultiplier = 2.5f;
-    public float lowFallMultiplier = 2.0f;
+    public float horizontalMovementModifier;
```

Comment:
I saw that this was modified from 0.25 to 25. Was this intentional?

Reply:
👍 Thanks for asking -- I should have explained that one.

To achieve a "pause", I set `Time.timeScale = 0f` so that the update loop would pass in `Time.deltaTime = 0` on each tick.
https://github.com/CleverGameDev/Cleverfall/pull/4/files#diff-a3b32680e093f6ac1eb1901b5df29674R28

I noticed that even when paused, the square would continue its horizontal movement. So I also made a modification here so that horizontal movement also used `Time.deltaTime`:

https://github.com/CleverGameDev/Cleverfall/pull/4/files#diff-2998edb257391c966c682433b9c08e2fL36-R36

However, this mean that the value of the movement variable was made much smaller. Previously, x was moved `xChange` per Update() tick, whereas afterwards it was moved `xChange * Time.deltaTime`. Since `Time.deltaTime` is a small float (say, about `1/60` if 60FPS), multiplying by 100 should result in about the same horizontal movement rate as before. I actually didn't think through this too carefully at the time, I just eyeballed it.